platform/linux-generic/odp_schedule_iquery.c

```diff
@@ -1296,7 +1296,14 @@ static void schedule_order_unlock(unsigned lock_index)
 	odp_atomic_store_rel_u64(ord_lock, thread_local.ordered.ctx + 1);
 }
 
-static unsigned schedule_max_ordered_locks(void)
+static void schedule_order_unlock_lock(uint32_t unlock_index,
+				       uint32_t lock_index)
+{
```

Comment:
Is this here due to a rebase error? Implementation of the new function should not be in this patch.